### DISCUSSION TITLE
Ikawa Import

### DISCUSSION THREAD

**Discussion by djbetterly** [artisan-roaster-scope/artisan#886] [Q&A] [answered]
I just imported an Ikawa roast, and I'm curious how I can turn off all the labels, or figure out a way to make it more legible.  I'm sure its something simple.

![ikawa-roast](https://user-images.githubusercontent.com/2065756/168898573-1fd762e4-512f-439f-953c-936e3d468217.png)
?

**Comment by roasterdave** [comment, marked as answer]
Go to Config>> Events>> Annotations tab.  Untick the events to hide their annotations.
